runtime/sema/checker.go

```diff
@@ -1890,6 +1894,14 @@ func (checker *Checker) predeclaredMembers(containerType Type) []*Member {
 
 	if compositeKindedType, ok := containerType.(CompositeKindedType); ok {
 
+		addPredeclaredMember(
```

Comment:
Why this can only be defined when `containerType` is a `CompositeKindedType`?

Why not moving this out of the above `if`

Reply:
Good point, I moved it out of the if-statement in 4e4fc03ac891c540b8c88af4b665001c5c088144